main.js

```diff
@@ -0,0 +1,147 @@
+const VALIDATE = (prop) => {
+    if (prop === undefined || prop === null) {
+        return true;
+    }
+    return false;
+}
+
+// 1. Cтворити функцію, яка створює та реєструє нову команду у автопробізі. Під час виклику функції потрібно передати дані про команду - назва команди, ім'я водія, рік народження водія, марка автомобіля, спонсор команди та чи водій допущений до автопробігу
+
+// 1st var
+const CAR_RACE_TEAM = (teamName, driverName, yearOfBirth, carBrand, sponsor, isAllowed) => {
+    if (VALIDATE(teamName) || VALIDATE(driverName) || VALIDATE(yearOfBirth) || VALIDATE(carBrand) || VALIDATE(sponsor) || VALIDATE(isAllowed) || isNaN(yearOfBirth)) {
```

Comment:
Ideally, you should validate not only `null` and `undefined` but also if the name is a string, if the year of birth is a number,